packages/calcite-components/src/components/input/input.e2e.ts

```diff
@@ -2064,7 +2064,77 @@ describe("calcite-input", () => {
           shadowSelector: `.${CSS.suffix}`,
           targetProp: "inlineSize",
         },
+        "--calcite-input-prefix-background-color": {
+          shadowSelector: `.${CSS.prefix}`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-input-suffix-background-color": {
+          shadowSelector: `.${CSS.suffix}`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-input-prefix-text-color": {
+          shadowSelector: `.${CSS.prefix}`,
+          targetProp: "color",
+        },
+        "--calcite-input-suffix-text-color": {
+          shadowSelector: `.${CSS.suffix}`,
+          targetProp: "color",
+        },
+        "--calcite-input-background-color": {
+          shadowSelector: `input`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-input-border-color": {
+          shadowSelector: `input`,
+          targetProp: "borderColor",
+        },
+        "--calcite-input-shadow": {
+          shadowSelector: `.${CSS.inputWrapper}`,
+          targetProp: "boxShadow",
+        },
+        // not sure how to target this ::placeholder style
```

Comment:
I don't think we can. We'll have to rely on the chromatic tests

Reply:
👍  removed this.